src/driver/WidgetDriver.ts

```diff
@@ -274,4 +274,27 @@ export abstract class WidgetDriver {
     ): Promise<ISearchUserDirectoryResult> {
         return Promise.resolve({ limited: false, results: [] });
     }
+
+    /**
+     * Get the config for the media repository.
+     * @returns Promise which resolves with an object containing the config.
+     */
+    public getMediaConfig(): Promise<{
```

Comment:
Would you mind declaring a seperate `interface` for the resolved return type, rather than defining the interface inline?

Reply:
I don't mind. Thanks for the hint. Fixed it.